core/src/test/scala/integration/kafka/api/AdminClientIntegrationTest.scala

```diff
@@ -759,7 +759,7 @@ class AdminClientIntegrationTest extends IntegrationTestHarness with Logging {
 
       val future = result.lowWatermarks().get(topicPartition)
       try {
-        lowWatermark = future.get(1000L, TimeUnit.MILLISECONDS).lowWatermark()
+        lowWatermark = future.get(5000L, TimeUnit.MILLISECONDS).lowWatermark()
```

Comment:
We should use the `get` method without a timeout. `waitUntilTrue` has its own timeout, so we don't need to have another one here.